internal/transform/scope-css.go

```diff
@@ -19,7 +19,7 @@ outer:
 		if n.DataAtom != a.Style {
 			continue
 		}
-		if hasTruthyAttr(n, "global") {
+		if hasTruthyAttr(n, "is:global") {
```

Comment:
Removed support for `global`, just supporting `is:global`. Is that what we want? It's minimal to support both.

Reply:
Added a migration warning for `global`, but still supporting both.